web/templates/stream_settings/stream_types.hbs

```diff
@@ -41,8 +41,8 @@
 </div>
 
 <div class="input-group new-style">
-    <label class="dropdown-title">{{t 'Who can unsubscribe others from this stream?'}}
-    </label>
+    <div class="dropdown-title">{{t 'Who can unsubscribe others from this stream?'}}
+    </div>
```

Comment:
I am not sure whether we want to change this to `div`, since the text still acts as a label for the dropdown setting.

Reply:
I know we do not use an `input` or `select` element here, but still worth pointing out in case we want feedback from others.

Reply:
Clicking on these `label` tags doesn't focus anywhere, which is not the expected behavior of label tags. Thus, using them seems logically incorrect. I replaced them with a more appropriate element `div`. I added a separate commit with the reasoning for this and also fixed the broken colors of the help links

Reply:
Why is the expected behavior of clicking on the label for a dropdown to focus something? 

(I do think it's probably a minor bug that this doesn't have a `for=` applied).

Reply:
Fixing this bug in recent commit.